### PR TITLE
[Release 1.5] Upgrade notice for new authentication APIs

### PR DESCRIPTION
**Pull request by diemtvu**
Upgrade notice for new authentication APIs and recommendation actions after upgrade.

Hold for references updated to provide links to the APIs.

### CODE REVIEW

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 1]
```diff
@@ -10,4 +10,17 @@ compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
 familiar with the use and operation of Istio 1.4.
 
-TBD
+## Authentication policy
+
+Istio 1.5 introduces [`PeerAuthentication`](/docs/reference/config/security/todo-link/) and [`RequestAuthentication`](/docs/reference/config/security/todo-linke/) that is going to replace alpha Authentication API. For more information how to use the new API, see the [authentication policy](/docs/tasks/security/authentication/authn-policy) tutorial.
```
Comment (elysdir) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
Istio 1.5 introduces [`PeerAuthentication`](/docs/reference/config/security/todo-link/) and [`RequestAuthentication`](/docs/reference/config/security/todo-link/), which are replacing the alpha version of the Authentication API. For more information about how to use the new API, see the [authentication policy](/docs/tasks/security/authentication/authn-policy) tutorial.
```

Comment (elysdir) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Also, can you clarify the timing? Does the new API replace the old API effective immediately? Is the old API now deprecated?

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 2]
```diff
@@ -10,4 +10,17 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+* After upgrading Istio, existing alpha authentication policies still in use.
+* You should start replacing alpha policies with the equivalent `PeerAuthentication` and `RequestAuthentication`. It is recommended to start with the most specific scope (workload-wide), then namespace-wide and finally mesh-wide.
+* Once the new API is added for workload/namespace/mesh, alpha API will be ignored in that scope and can be safely remove.
+* After finish migrating to new authentication APIs, you can delete all alpha policies, if you haven't done so, with this command:
```
Comment (elysdir) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion

* After you upgrade Istio, your alpha authentication policies remain in place until you replace them with the equivalent `PeerAuthentication` and `RequestAuthentication`. We recommend starting with workload-wide (the most specific scope), then namespace-wide, and finally mesh-wide.
* After you replace policies for workload, namespace, and mesh, you can safely remove the alpha authentication policies. To delete the alpha policies, use this command:
```

**Comment by diemtvu** [comment]
I rewrite the first bullet to mention the old API is still in used until the new one created. Ptal. Thanks.

**Comment by duderino** [comment]
@diemtvu says this is blocked on https://github.com/istio/istio.io/pull/6419

**Comment by diemtvu** [comment]
This is now blocked on the missing links caused by https://github.com/istio/istio.io/pull/6740 and https://github.com/istio/istio.io/pull/6727 (all presumit tests are failing at the moment).

File: content/en/about/feature-stages/index.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Service-to-service mutual TLS](/docs/concepts/security/#mutual-tls-authentication)         | Stable
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
-| [SDS based key and certificate provisioning](/docs/concepts/security/#the-istio-agent-in-kubernetes)  | Stable
+| [SDS based key and certificate provisioning](/docs/concepts/security/#pki)  | Stable
```
Comment (diemtvu) [inline content/en/about/feature-stages/index.md]: + @myidpt FYI, I change the link to this so this PR can be merged. Please take care of this later when you have chance.

**Pull request merged**